### ISSUE TITLE
chrome dropdown not showing full window

### ISSUE THREAD

**Issue by manarchist8691** [EFForg/privacybadger#1113]
there's no way to scroll to the options at the bottom on this site. at least some other sites work ok.
![screenshot 2016-12-25 12 49 19](https://cloud.githubusercontent.com/assets/24766112/21472319/db17c9a8-caa1-11e6-99ef-932d0f392996.png)

**Comment by cooperq** [comment]
Do you happen to have page zoom set to something other than 100% in chrome settings? That seems to cause this issue.

**Comment by manarchist8691** [comment]
Yes, I default to 125%. Guess I can change that if need be. Thanks.

**Comment by cooperq** [comment]
That would be a good workaround for now. We should probably support larger font sizes long term though.

**Comment by lemnis** [comment]
This should not happen anymore since #1516, it should now add a scrollbar to the popup. @manarchist8691 , can you confirm this?

**Comment by realnc** [comment]
It now has scrollbars for me.
